hadoop-ozone/recon/src/test/java/org/apache/hadoop/ozone/recon/api/TestNSSummaryEndpointWithLegacy.java

```diff
@@ -134,6 +134,7 @@ public class TestNSSummaryEndpointWithLegacy {
   private static final String BUCKET_TWO = "bucket2";
   private static final String BUCKET_THREE = "bucket3";
   private static final String BUCKET_FOUR = "bucket4";
+  private static final String BUCKET_FIVE = "bucket5";
```

Comment:
Are we asserting anything in the test class `TestNSSummaryEndpointWithLegacy`? We have created a bunch of new files and directories but have not asserted anything yet.

Reply:
> Are we asserting anything in the test class `TestNSSummaryEndpointWithLegacy`? We have created a bunch of new files and directories but have not asserted anything yet.

I added the test case for both legacy and obs buckets in `TestNSSummaryEndpointWithOBSAndLegacy` class. Will remove the tree structure from here.